tests/RegistryPageFunctionalTest.php

```diff
@@ -22,12 +24,12 @@ class RegistryPageFunctionalTest extends FunctionalTest
         RegistryPageTestPage::class
     ];
 
-    protected static $use_draft_site = true;
```

Comment:
Removing because FuntionalTest.use_draft_site is deprecated. To fix the unit test I just publish all the pages.